.github/workflows/compose-maven-verify.yaml

```diff
@@ -122,6 +122,7 @@ jobs:
               CI_REPO="ci-management"
           fi
           wget -q -O settings.xml "https://raw.githubusercontent.com/${{ vars.ORGANIZATION }}/${CI_REPO}/master/jenkins-config/managed-config-files/globalMavenSettings/global-settings/content"
+          sed -i "s#\^##" settings.xml
```

Comment:
Since you want to explicitly remove the ^ from ^${ please make sure that you're specifying that in your regex.

The sed really should look more like this:

`sed -i 's#\^${#${#' settings.xml`

Reply:
Thanks, resolved!